vue/src/modules/Workspace/view/articles/WorkspaceArticles.vue

```diff
@@ -35,12 +35,15 @@
                        :target="`copy-tooltip-${article.id}`" :title="$t('Copied!')" placement="auto"></b-tooltip>
             <i :id="`copy-tooltip-${article.id}`" class="mr-3 far fa-copy text-primary hover-pointer"
                @click="onCopyUrlClick(article)"></i>
```

Comment:
@skavt Copy url does not work because the urls were changed. Please copy view url.